test-result-summary-client/src/Build/GitNewIssue.jsx

```diff
@@ -32,6 +33,13 @@ export default class GitNewissue extends Component {
         const testData = await fetchTestData.json();
         const { testName, duration, testResult } = testData;
 
+        // fetch test output
+        const fetchTestOutput = await fetch(`/api/getOutputById?id=${testData.testOutputId}`, {
+            method: 'get'
+        });
+        const TestOutput = await fetchTestOutput.json();
+        const failureOutput = parseTestOutput(Object.values(TestOutput)[0]);
```

Comment:
Instead of calling `getOutputById` API, the logic of `parseTestOutput` should be a server side API. This may be used by others in the future.